### ISSUE TITLE
[BitBucket] "Set up Code Mapping" modal in Issue Details

### ISSUE THREAD

**Issue by InterstellarStella** [getsentry/sentry#68600]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

1. Set up the BitBucket integration
2. Open an issue with an event that has working source maps

### Expected Result

There is nothing in the stack trace prompting us to set up code mappings.

### Actual Result

We see the "Set up Code Mapping" modal in the stack trace. This opens a window that prompts us to paste a URL for the path to the file from BitBucket. However, currently this only works for GitLab and GitHub, and we'd have to set up the code mappings manually through the integration settings instead. This leads to a "We don't have access to that source code repo" error banner once clicking "Save", or doesn't do anything at all.

We should either remove this modal when we detect that only the BitBucket integration is set up, or add the functionality for BitBucket too.

### Product Area

Settings - Integrations

**Comment by cathteng** [comment]
We have docs for [setting up codemappings with Bitbucket](https://docs.sentry.io/product/integrations/source-code-mgmt/bitbucket/), so it theoretically should work. It appears that the modal in the stack trace might not be working properly if you are able to set up the code mapping through the integration settings. I will ask issues team if they know anything about this.

**Comment by malwilley** [comment]
@InterstellarStella you're right! We should at the very least display some help content in that modal rather than the form when we don't support the source code provider. Will place on our backlog.

The workaround for now is to go to the docs page that @cathteng linked ([specifically this section](https://docs.sentry.io/product/integrations/source-code-mgmt/bitbucket/#stack-trace-linking)). While automatic code mapping generation is not supported for Bitbucket, users can still set it up themselves using that info.
